lib/arel/visitors/sqlserver.rb

```diff
@@ -11,13 +11,18 @@ class SQLServer < Arel::Visitors::ToSql
 
       private
 
-      # SQLServer ToSql/Visitor (Overides)
+      # SQLServer ToSql/Visitor (Overrides)
 
-      def visit_Arel_Nodes_BindParam o, collector
-        collector.add_bind(o.value) { |i| "@#{i - 1}" }
+      BIND_BLOCK = proc { |i| "@#{i - 1}" }
+      private_constant :BIND_BLOCK
+
+      def bind_block; BIND_BLOCK; end
+
+      def visit_Arel_Nodes_BindParam(o, collector)
```

Comment:
With `bind_block` introduction, we can remove `visit_Arel_Nodes_BindParam` now that it has the same definition as the parent Abstract adapter.

Reply:
Removed.